### PR TITLE
configure every replicant in test

### PR DESCRIPTION
**Pull request by dorinhogea**
Need to run PUT on every node to be able to discover comdb2db replacement.

### CODE REVIEW

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: Success &#10003;.<br>Smoke testing: Error &#x26A0;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>2/476 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>sc_timepart_logicalsc_generated<br>incoh_remsql</p>

File: tests/tools/foreigndbconfig.c [hunk 1]
```diff
@@ -33,22 +35,27 @@ int main (int argc, char *argv[])
             case 'c':
                 config = optarg;
                 break;
+            case 'm':
+                mach = optarg;
```
Comment (markhannum) [inline tests/tools/foreigndbconfig.c:39]: break;

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: Success &#10003;.<br>Smoke testing: Error &#x26A0;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>3/476 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>sc_transactional_rowlocks_generated<br>queuedb_rollover<br>sc_timepart_logicalsc_generated</p>

**Comment by dorinhogea** [comment]
Thank you Salil!

**Pull request merged**